package.json

```diff
@@ -13,51 +13,52 @@
     "format": "prettier --write src/frontend"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/jest-dom": "^5.14.1",
     "@trust/webcrypto": "^0.9.2",
-    "@types/jest": "^24.9.1",
-    "@types/selenium-webdriver": "^4.0.12",
-    "@types/text-encoding": "^0.0.35",
-    "@typescript-eslint/eslint-plugin": "^4.22.0",
-    "@typescript-eslint/parser": "^4.22.1",
+    "@types/jest": "^26.0.24",
+    "@types/selenium-webdriver": "^4.0.15",
+    "@types/text-encoding": "^0.0.36",
+    "@typescript-eslint/eslint-plugin": "^4.28.4",
+    "@typescript-eslint/parser": "^4.28.4",
     "assert": "2.0.0",
     "buffer": "6.0.3",
-    "chromedriver": "^90.0.0",
-    "compression-webpack-plugin": "^7.1.2",
-    "copy-webpack-plugin": "^8.1.1",
-    "css-loader": "^5.2.1",
-    "dotenv": "^8.2.0",
-    "eslint": "^7.25.0",
+    "chromedriver": "^91.0.1",
+    "compression-webpack-plugin": "^8.0.1",
+    "copy-webpack-plugin": "^9.0.1",
+    "css-loader": "^6.2.0",
+    "dotenv": "^10.0.0",
+    "eslint": "^7.31.0",
     "events": "3.3.0",
     "file-loader": "^6.2.0",
-    "html-webpack-plugin": "5.3.1",
-    "jest": "^26.6.3",
-    "jest-cli": "^26.6.3",
-    "prettier": "2.2.1",
+    "html-webpack-plugin": "5.3.2",
+    "jest": "^27.0.6",
+    "jest-cli": "^27.0.6",
+    "prettier": "2.3.2",
     "process": "0.11.10",
-    "selenium-webdriver": "^4.0.0-beta.3",
+    "selenium-webdriver": "^4.0.0-beta.4",
     "stream-browserify": "3.0.0",
-    "style-loader": "^2.0.0",
-    "terser-webpack-plugin": "5.1.1",
-    "ts-jest": "^26.5.4",
-    "ts-loader": "8.1.0",
-    "ts-node": "^9.1.1",
-    "typescript": "^4.2.4",
+    "style-loader": "^3.2.1",
+    "terser-webpack-plugin": "5.1.4",
+    "ts-jest": "^27.0.3",
+    "ts-loader": "9.2.3",
+    "ts-node": "^10.1.0",
+    "typescript": "^4.3.5",
     "url-loader": "^4.1.1",
-    "util": "0.12.3",
-    "webpack": "5.24.4",
-    "webpack-cli": "4.5.0",
+    "util": "0.12.4",
+    "webpack": "5.45.1",
+    "webpack-cli": "4.7.2",
```

Comment:
`webpack-dev-server` depends on vulnerable version of chokidar -> glob-parent 5.1.2. The fix suggested by npm audit is downgrading to 1.16.5, which is probably not an option. Also, given that this is only a dev dependency, I suggest we don't act on this now.